hello-world/views/first_view.pug

```diff
@@ -0,0 +1,6 @@
+doctype
```

Comment:
@psinha20 .pug file is necessary for the project ? If not, then please remove this.